controllers/choose_removals.go

```diff
@@ -69,6 +69,7 @@ func (c chooseRemovals) reconcile(ctx context.Context, r *FoundationDBClusterRec
 	}
 
 	localityMap := make(map[string]locality.Info)
+	// should be cross-DC; this seems to choose what processes to keep based on cross-dc data
```

Comment:
We can add the filter here to limit the range over the operator DC. The operator will (and can) only remove process groups in the DC that the operator mages (there operator is not able to read the FoundationDBCluster resource in another Kubernetes cluster).

Reply:
You are correct, it can't remove things outside the DC - though I am a bit tripped up because `ChooseDistributedProcesses` accounts for DCID in its logic, and am trying to thoroughly understand the implications of only handing it a set of processes that are within the DC before I fix this

Reply:
Discussed offline, the `ChooseDistributedProcesses` does not matter because the `reconcile` for removals already filters processes outside the DC in the line `for _, processGroup := range cluster.Status.ProcessGroupsByProcessClass(processClass) {` .  
I will add a filter here to make it more explicit